app/lib/frontend/templates/views/publisher/publisher_list.mustache

```diff
@@ -0,0 +1,28 @@
+{{! Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+    for details. All rights reserved. Use of this source code is governed by a
+    BSD-style license that can be found in the LICENSE file. }}
+
+<h2>Publishers</h2>
+
+{{#has_publishers}}
+<ul class="package-list">
+  {{#publishers}}
+  <li class="list-item">
+    <h3 class="title"><a href="{{& url}}">{{publisher_id}}</a></h3>
+    <p class="metadata">
+      Registered on {{short_created}}.
+    </p>
```

Comment:
Showing number of members and number packages would be too expensive, right?

Reply:
We may do it, depending on where we put the aggregates.